routes/userWeb.js

```diff
@@ -1,37 +1,10 @@
 const express = require('express')
 const router = express.Router();
-const userWebController = require('../controller/userWebController');
 
-router.get('/test', userWebController.test);
+const authrouter = require('./auth');
+const studentrouter = require('./student'); 
+const managerrouter = require('./manager'); 
 
-router.post('/login', userWebController.userLogin);
-router.post('/loginVerify', userWebController.loginVerify);
-
-router.post('/studentLoginVerify', userWebController.studentVerify);
-router.post('/studentRegister', userWebController.studentRegister);
-router.post('/studentEditData', userWebController.studentEditData);
-router.post('/getStudentPlacements', userWebController.getStudentPlacements);
-
-router.post('/registerOfficial', userWebController.registerOfficial);
-router.post('/addStudent', userWebController.addStudent);
-router.get('/getRegisteredOfficials', userWebController.getRegisteredOfficials);
-router.post('/toggleOfficialStatus', userWebController.toggleOfficialAccountStatus);
-
-router.post('/forgotPassword', userWebController.forgotPassword);
-router.post('/resetPasswordVerify', userWebController.resetPasswordVerify);
-router.post('/resetPassword', userWebController.resetPassword);
-
-router.post('/addCompany', userWebController.addCompany);
-router.get('/getCompanies', userWebController.getCompanies);
-router.post('/addPlacementData', userWebController.addPlacementData);
-router.post('/editPlacementDataById', userWebController.editPlacementDataById);
-
-router.get('/getCompanyHireData',userWebController.getCompanyHireData);
-router.post('/getCompanyHireDataByBatch',userWebController.getCompanyHireDatabyBatch);
-router.post('/getCompanyHireDataById',userWebController.getCompanyHireDataById);
-
-router.post('/getAllStudentData',userWebController.getAllStudentData);
-router.get('/getTopFivePlacements',userWebController.getTop5Placements);
-router.post('/getAllPlacedStudentData',userWebController.getAllPlacedStudentsData);
-
-module.exports = router;
\ No newline at end of file
+router.use('/auth', authrouter); 
+router.use('/student', studentrouter); 
+router.use('/manager', managerrouter); 
```

Comment:
move the base routes to `index.js` and delete this file.

Reply:
## Added these lines directly to index.js..

![image](https://github.com/user-attachments/assets/fba955a7-f711-4359-b257-486c4240d1a4)

![image](https://github.com/user-attachments/assets/d5371f3d-d79a-4a16-bb47-2b27cd5a8f91)